include/triton/codegen/analysis/layout.h

```diff
@@ -109,21 +109,89 @@ class distributed_layout: public data_layout{
 };
 
 class mma_layout: public distributed_layout {
+public:
+  enum TensorCoreType : uint8_t {
+    // floating-point tensor core instr
+    FP32_FP16_FP16_FP32 = 0, // default
+    FP32_BF16_BF16_FP32, // Not implemented
+    FP32_TF32_TF32_FP32, // Not implemented
```

Comment:
might want to change the comments there ;)

Reply:
Indeed!